examples/convert_eeg_to_bids.py

```diff
@@ -85,7 +85,7 @@
 # We will do exactly that in the next step.
 
 ###############################################################################
-# Converting to BIDS
+# Convert to BIDS
 # ------------------
```

Comment:
```suggestion
# Convert to BIDS
# ---------------
```

@sappelhoff Just testing your patience here… aaargh

Reply:
:rofl: are you trying to push your commit count?